src/main/webapp/script.js

```diff
@@ -82,12 +84,24 @@ function drawRegionsMap(censusDataArray) {
 // createDataArray takes in the data array returned by the census API
 // and reformats it into a data table for the visualization API.
 function createDataArray(censusDataArray) {
-  const vizDataArray = [];
-  censusDataArray.forEach((state) => {
-    vizDataArray.push([state[0], state[1]/1000]);
-  });
+  const vizDataArray = []
+  if (censusDataArray[0].length == 4) {
```

Comment:
Like I mentioned in another comment should be triple equals

Reply:
fixed!